### PR TITLE
Auth Service: Convert to Helm

### PR DESCRIPTION
**Pull request by edunham**
Need to figure out that json-e list and also the right way to handle all those locals (current strategy of ignoring them is obviously great for writing boilerplate and awful for emitting a working service)

### CODE REVIEW

File: infrastructure/k8s/services/auth.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+project_name: "taskcluster-auth"
+secrets:
+  aws_access_key_id: ".Values.awsAccessKeyId"
+  aws_region: ".Values.awsRegion"
+  aws_secret_access_key: ".Values.awsSecretAccessKey"
+  azure_account: ".Values.tfAzureAccount"
+  azure_accounts: "" # FIXME this was "${jsonencode(map( "${azurerm_storage_account.base.name}", "${azurerm_storage_account.base.primary_access_key}",))}"
```
Comment (sciurus) [inline infrastructure/k8s/services/auth.yaml]: As a first pass I'd put the json for `{account_name: access_key}` as a string value in sops and look it up here.

File: infrastructure/k8s/services/auth.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  monitoring_enable: "true"
+  node_env: "production"
+  owner_email: "bstack@mozilla.com" #FIXME should probably be cloudops
+  profile: "production"
+  publish_metadata: "false"
+  pulse_hostname: ".Values.pulseHostname"
+  pulse_password: ".Values.authPulsePassword"
+  pulse_username: ".Values.authPulseUsername"
+  pulse_vhost: ".Values.pulseVhost"
+  sentry_api_key: ".Values.sentryApiKey"
```
Comment (sciurus) [inline infrastructure/k8s/services/auth.yaml]: Can you check if they still use sentry?

**Review by sciurus — CHANGES_REQUESTED** [review]
As a first pass at `static_clients` I would just build the JSON for that list and stick it in sops as a string then look it ip.

**Comment by edunham** [comment]
r+ from me on this -- let's see how the blob in sops goes!

**Comment by sciurus** [comment]
I think this still needs some work, but it's starting up, so I'll consider that mergeable.

**Pull request merged**